etc/profile-a-l/gtk-straw-viewer.profile

```diff
@@ -8,7 +8,5 @@ include gtk-straw-viewer.local
 
 ignore quiet
 
-include whitelist-runuser-common.inc
```

Comment:
Why?

Reply:
the 3 gtk-xxxx-viewer are basically the same. They differ in how they interact with youtube.com . Either all should have wruc, or none should have it.

Reply:
That's a working hardening option, removing it just be because "Either all should have wruc, or none should have it" is not reasonable.

Reply:
originally i added wruc to  gtk-pipe-viewer, just by copying from these two. You complained and now i rm them from all. I don't see anything broken in gtk-pipe-viewer without it. xxxx-viewer are the cli variants.

Reply:
> You complained

I asked why only gtk-pipe-viewer has it and the others not. (https://github.com/netblue30/firejail/pull/4128#discussion_r599012588)

>  I don't see anything broken in gtk-pipe-viewer without it.

How can something break if you remove it?

> xxxx-viewer are the cli variants.

There we could probably even `blacklist ${RUNUSER}` if we had no restriction to be able to start media players.

Reply:
originally only the gui versions had this. I just copied the format.

i add "include whitelist-runuser-common.inc" in youtube-viewers-common.profile?
They all inherit from youtube-viewers-common.profile

Reply:
> i add "include whitelist-runuser-common.inc" in youtube-viewers-common.profile?

If it works there, we should add it there.

Reply:
done